crates/biome_html_analyze/tests/specs/a11y/useAriaPropsSupportedByRole/valid.html

```
@@ -0,0 +1,93 @@
+<!-- should not generate diagnostics -->
+<div></div>
+<div id="main"></div>
+<div role=""></div>
+<a href="#" aria-expanded="true"></a>
+<a href="#" aria-atomic="true"></a>
+<a href="#" aria-busy="true"></a>>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)